main.js

```diff
@@ -1,7 +1,8 @@
 (function(){
   var node = document.querySelector("div");
   var app = Elm.Main.embed(node, {
-      eventId: window.location.href.split('/events/')[1] ? window.location.href.split('/events/')[1].split('/')[0] : ""
+      eventId: window.location.href.split('/events/')[1] ? window.location.href.split('/events/')[1].split('/')[0] : "",
```

Comment:
does the npm module still work if `/events/` isn't in the url? Could be a case of overly tight coupling.